src/views/AuthoritiesSearch/AuthoritiesSearch.test.js

```diff
@@ -222,10 +222,10 @@ describe('Given AuthoritiesSearch', () => {
       fireEvent.click(getByRole('button', { name: 'stripes-components.paneMenuActionsToggleLabel' }));
 
       const marcAuthorityHeadingsButton = getByRole('button', { name: 'ui-marc-authorities.reports.marcAuthorityHeadings' });
-      const failedUpdatesButton = getByRole('button', { name: 'ui-marc-authorities.reports.failedUpdates' });
+      // const failedUpdatesButton = getByRole('button', { name: 'ui-marc-authorities.reports.failedUpdates' });
```

Comment:
I think it would be better to add to the comment the reason why it was commented out. On all the commented out places in the files. So as not to accidentally delete it in the future.